### PR TITLE
Include coins and hours in /explorer/address inputs

### PR DESCRIPTION
**Pull request by therealssj**
Resolves: #866

### CODE REVIEW

**Comment by therealssj** [comment]
fixed conflict

**Comment by gz-c** [comment]
Do you remember the issue/PR in explorer that needed this info?

File: src/visor/readable.go [hunk 1]
```diff
@@ -145,10 +147,12 @@ func NewReadableTransactionOutput(t *coin.TransactionOutput, txid cipher.SHA256)
 }
 
 // NewReadableTransactionInput creates readable transaction input
-func NewReadableTransactionInput(uxID string, ownerAddress string) ReadableTransactionInput {
+func NewReadableTransactionInput(uxID, ownerAddress, coins, hours string) ReadableTransactionInput {
```
Comment (gz-c) [inline src/visor/readable.go]: Probably better to have `coins, hours uint64` and do the string conversions in here

Comment (therealssj) [reply src/visor/readable.go]: But then will have to do error handling for `coins` inside the function

Comment (gz-c) [reply src/visor/readable.go]: that's fine

**Pull request merged**